module/Application/view/layout/layout.phtml

```diff
@@ -12,15 +12,15 @@
 
         <!-- Le styles -->
         <?= $this->headLink(['rel' => 'shortcut icon', 'type' => 'image/vnd.microsoft.icon', 'href' => $this->basePath() . '/img/favicon.ico'])
-                        ->prependStylesheet($this->basePath('css/style.css'))
-                        ->prependStylesheet($this->basePath('css/bootstrap-theme.min.css'))
-                        ->prependStylesheet($this->basePath('css/bootstrap.min.css'))
+            ->appendStylesheet($this->basePath('css/bootstrap.min.css'))
+            ->appendStylesheet($this->basePath('css/bootstrap-theme.min.css'))
+            ->appendStylesheet($this->basePath('css/style.css'))
         ?>
 
         <!-- Scripts -->
         <?= $this->headScript()
-            ->prependFile($this->basePath('js/bootstrap.min.js'))
-            ->prependFile($this->basePath('js/jquery-2.2.4.min.js'))
+            ->appendFile($this->basePath('js/jquery-2.2.4.min.js'))
+            ->appendFile($this->basePath('js/bootstrap.min.js'))
```

Comment:
In both blocks, `prepend` is correct, as rendering is done depth-first, and typically your individual pages will have *additional* CSS/JS that you want to *override* the base styles and scripts. As such, I'll revert these on merge.